node/overseer/src/lib.rs

```diff
@@ -1318,10 +1318,9 @@ where
 	/// Run the `Overseer`.
 	#[tracing::instrument(skip(self), fields(subsystem = LOG_TARGET))]
 	pub async fn run(mut self) -> SubsystemResult<()> {
-		let leaves = std::mem::take(&mut self.leaves);
 		let mut update = ActiveLeavesUpdate::default();
 
-		for (hash, number) in leaves.into_iter() {
```

Comment:
any reason to reverse the order here?

Reply:
No, but I also don't see any message about a forced order.